custom_components/mail_and_packages/helpers.py

```diff
@@ -381,8 +381,10 @@ def fetch(
         delivering = fetch(hass, config, account, data, f"{prefix}_delivering")
         delivered = fetch(hass, config, account, data, f"{prefix}_delivered")
         count[sensor] = delivering + delivered
-    elif "_delivering" in sensor:
+    elif "_delivering" in sensor or "_exception" in sensor or "_processing" in sensor:
         prefix = sensor.replace("_delivering", "")
+        prefix = sensor.replace("_exception", "")
+        prefix = sensor.replace("_processing", "")
```

Comment:
This change isn't needed exceptions are already handled in the function.

Reply:
I don't see this anywhere in the function. This compliments the two new states I added, processing and exception to be able to check which parcels are in those states. Exception state is important to know what parcels need attention

Reply:
Exceptions are handled when they fall out of the bottom of the `elif` branches.

https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/blob/dev/custom_components/mail_and_packages/helpers.py#L405-408